metloom/variables.py

```diff
@@ -346,3 +346,82 @@ class MetNorwayVariables(VariableBase):
         " Each tip is registered along with the time stamp for the tip."
         " This is the basis for calcutation of precipitation sum per minute"
     )
+
+
+class SnowExVariables(VariableBase):
+    """
+    Variables for SnowEx met stations data, refer to user guide for adding more
+    variables
+
+    Metadata:
+    https://nsidc.org/sites/default/files/documents/user-guide/snex_met-v001-userguide.pdf
+    """
+
+    TEMP_20FT = InstrumentDescription(
+        'AirTC_20ft_Avg', "AIR TEMP @20ft",
+        description="Air temperature measured at 20 ft tower levelin deg C")
+
+    TEMP_10FT = InstrumentDescription(
+        'AirTC_10ft_Avg', "AIR TEMP @10ft",
+        description="Air temperature measured at 10 ft tower levelin deg C")
+
+    UPSHORTWAVE = InstrumentDescription(
+        "SUp_Avg", "UPWARD SHORTWAVE RADIATION",
+        description="Shortwave radiation measured with upward-facing sensor",
+        instrument="CNR4 Net Radiometer"
+    )
+    DOWNSHORTWAVE = InstrumentDescription(
+        "SDn_Avg", "DOWNWARD SHORTWAVE RADIATION",
+        description="Shortwave radiation measured with downward-facing sensor",
+        instrument="CNR4 Net Radiometer"
+    )
+    SNOWDEPTH = InstrumentDescription(
+        "SnowDepthFilter(m)", "SNOW DEPTH",
```

Comment:
Can you change SNOW DEPTH to SNOWDEPTH for consistency?
```suggestion
        "SnowDepthFilter(m)", "SNOWDEPTH",
```